<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Обучение</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1131</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="inputValuesBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>141</width>
         <height>251</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Входные значения</string>
       </property>
       <widget class="QTextEdit" name="inputVal1">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="inputVal2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="inputVal3">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="inputVal4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>170</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="inputVal5">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>220</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="targetValBox">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>151</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Целевое значение</string>
       </property>
       <widget class="QTextEdit" name="targetVal">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="debugLabel4Learn">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>331</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="clearBtn">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>310</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Сбросить параметры</string>
       </property>
      </widget>
      <widget class="QPushButton" name="learnStartBtn">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>170</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Старт</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>230</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QToolButton" name="loadFileBtn">
       <property name="geometry">
        <rect>
         <x>1100</x>
         <y>10</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="learnFAQBtn">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>10</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Справка</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Прогон</string>
     </attribute>
     <widget class="QGroupBox" name="defQueryBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>991</width>
        <height>371</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="targetValBox_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>631</width>
         <height>201</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Выходное значение</string>
       </property>
       <widget class="QLabel" name="outputLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>591</width>
          <height>151</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Unispace</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="inputValuesBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>141</width>
         <height>261</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Входные значения</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QTextEdit" name="inputVal1_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="inputVal2_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="inputVal3_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>120</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="inputVal4_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>170</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="inputVal5_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>220</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="queryBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>310</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Прогнать</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="debugLabel4Query">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>270</y>
         <width>511</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="queryFAQBtn">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>10</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Справка</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxLink">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1131</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Количество связей между входным
                и скрытым слоем</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="hNodesIn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Количество связей между скрытым
                и выходным слоем</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="oNodesIn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="chLinksBtn">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>210</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
      <widget class="QLabel" name="debugLabel4Links">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>241</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>231</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Количество циклов обучения</string>
       </property>
       <widget class="QTextEdit" name="loopCount">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="epochsGroupBox">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>110</y>
         <width>231</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Количество эпох</string>
       </property>
       <widget class="QTextEdit" name="epochsCount">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="epochsLoopsBtn">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>210</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>30</y>
         <width>231</width>
         <height>80</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Коэффициент обучения</string>
       </property>
       <widget class="QTextEdit" name="learnRateBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>781</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>20</width>
         <height>241</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>781</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>20</y>
         <width>20</width>
         <height>241</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="setingsFAQBtn">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>10</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Справка</string>
       </property>
      </widget>
      <widget class="Line" name="line_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>20</width>
         <height>241</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="loadFromFile"/>
    <addaction name="currentParamBtn"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loadFromFile">
   <property name="text">
    <string>Загрузить внешний файл(.txt)</string>
   </property>
  </action>
  <action name="currentParamBtn">
   <property name="text">
    <string>Просмотреть текущие параметры</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
